### ISSUE TITLE
[no-use-before-declare] false positive when multiple declaration

### ISSUE THREAD

**Issue by kuza55** [palantir/tslint#153]
There seems to be an issue with no-user-before-declare having a false positive when there are multiple declarations of a variable, e.g. this code:

```
            if (something) {
                var defined = 1;
            } else {
                var defined;
            }
```

Produces this output:
```
$ ../node_modules/tslint/bin/tslint -f test.ts
test.ts[2, 21]: variable 'defined' used before declaration
```
